modules/combined/tests/eigenstrain/composite.i

```diff
@@ -5,6 +5,26 @@
   ny = 5
 []
 
+[GlobalParams]
```

Comment:
To get nonzero values for the aux variables of interest, I do need to solve the problem with the mechanics additions.  Perhaps it is the `solve = false` in the original test that keeps the `_step_zero` change to `false` from being evaluated?